diff --git a/projects/fmcjesdadc1/kc705/system_bd.tcl b/projects/fmcjesdadc1/kc705/system_bd.tcl
index baefd343..3a907632 100644
--- a/projects/fmcjesdadc1/kc705/system_bd.tcl
+++ b/projects/fmcjesdadc1/kc705/system_bd.tcl
@@ -1,6 +1,7 @@
 
 source $ad_hdl_dir/projects/common/kc705/kc705_system_bd.tcl
 source ../common/fmcjesdadc1_bd.tcl
+source ./esther_intfc_bd.tcl
 
 #system ID
 ad_ip_parameter axi_sysid_0 CONFIG.ROM_ADDR_BITS 9
